python/tidy/tidy.py

```diff
@@ -232,7 +232,8 @@ def next(self) -> str:
 
 
 def filter_file(file_name: str) -> bool:
-    if any(file_name.startswith(ignored_file) for ignored_file in config["ignore"]["files"]):
+    current_file = f"./{relative_path(file_name)}"
```

Comment:
We should use `os.path.join` so we don't hardcode the seperators (it is `\` for Window). Also, are there any downsides to making the paths returned by `FileList` relative instead of fixing only this method where it is consumed?